app/src/main/java/com/example/vybe/VibeFactory.java

```diff
@@ -49,4 +49,28 @@ else if (selectedVibe.equals("surprised")) {
 
         return null;
     }
```

Comment:
Reminder to remove getVibe(string selectedVibe) once we shifted all usages to getVibe(int selectedVibe)